simplq/src/components/common/LoadingIndicator/loading.module.scss

```diff
@@ -3,5 +3,5 @@
 .main {
   @include center-vertically();
   @include center-horizontally();
-  min-height: 100vh;
+  min-height: 30vh;
```

Comment:
This change makes sense. 

Now I'm wondering if we can completely remove this min-height here. @mathkruger What do you think?

Reply:
I tested without the "min-height" and, on some pages, it didn't look any good. I tried some values and it worked on every page where I could see this component.